app/views/map/show.html.erb

```diff
@@ -1,9 +1,18 @@
-<iframe
-  id="idIframe"
-  onload="iframeLoaded"
-  width="100%"
-  height="100%"
-  frameborder="0" style="border:0"
-  src="https://www.google.com/maps/embed/v1/search?key=<%= ENV['GOOGLE_MAP_KEY'] %>
-    &<%= @search_query_url %>" allowfullscreen>
-</iframe>
+<div id="mapid"></div>
+
+<script>
+  var mymap = L.map('mapid').setView([<%= @lat %>, <%= @lon %>], 13);
+
+  L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}', {
+    attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+    maxZoom: 18,
+    id: 'mapbox.streets',
+    accessToken: 'pk.eyJ1IjoibWFyeWFncmFjZSIsImEiOiJjanM0eG9qMmIwOXlhNDVuaTNuOXMzcXd1In0.Y3D7BRb8btkvg5JV_6PZaQ'
```

Comment:
we need this to be an ENV, had trouble accessing envs from this js method. maybe wait to merge until we figure this out